### ISSUE TITLE
Talk: TypeError after opening "Apps" menu in Leap

### ISSUE THREAD

**Issue by bonbud-macryg** [tloncorp/landscape-apps#2363]
```
TypeError: Cannot destructure property 'title' of 'e' as it is undefined.
    at ae (http://100.93.206.31/apps/talk/assets/grid.352f43e4.js:1:1610)
    at Ko (http://100.93.206.31/apps/talk/assets/react-beautiful-dnd.a94a7f90.js:19:52865)
    at ea (http://100.93.206.31/apps/talk/assets/react-beautiful-dnd.a94a7f90.js:23:8695)
    at Js (http://100.93.206.31/apps/talk/assets/react-beautiful-dnd.a94a7f90.js:23:935)
    at Qf (http://100.93.206.31/apps/talk/assets/react-beautiful-dnd.a94a7f90.js:23:866)
    at sn (http://100.93.206.31/apps/talk/assets/react-beautiful-dnd.a94a7f90.js:23:725)
    at mo (http://100.93.206.31/apps/talk/assets/react-beautiful-dnd.a94a7f90.js:21:10863)
    at http://100.93.206.31/apps/talk/assets/react-beautiful-dnd.a94a7f90.js:19:38697
    at e.unstable_runWithPriority (http://100.93.206.31/apps/talk/assets/react-beautiful-dnd.a94a7f90.js:8:3835)
    at st (http://100.93.206.31/apps/talk/assets/react-beautiful-dnd.a94a7f90.js:19:38475)
```

**Comment by bonbud-macryg** [comment]
The same button works fine in Groups.

**Comment by patosullivan** [comment]
Can you provide the output of `+vats`?  I'm wondering if you might have an app in a weird state.

**Comment by bonbud-macryg** [comment]
```
> +vats
nonexistent desk: %urdl-user
  %bitcoin
  /sys/kelvin:      [%zuse 417] [%zuse 416] [%zuse 415]
  base hash:        0v1o.0u604.fpjjg.j4ave.tsa1d.6uv06.ho51p.lkklh.er24d.kkf24.7fmh8
  %cz hash:         0v1o.0u604.fpjjg.j4ave.tsa1d.6uv06.ho51p.lkklh.er24d.kkf24.7fmh8
  app status:       suspended
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          local
  source ship:      ~
  source desk:      ~
  source aeon:      ~
  kids desk:        ~
  pending updates:  ~
::
  %verse
  /sys/kelvin:      [%zuse 417] [%zuse 416] [%zuse 415] [%zuse 414]
  base hash:        0v1f.hdig7.uavpc.3r4u4.kuioi.7tjfp.9lr4k.3p3i3.m6mc5.k14jf.m4kn2
  %cz hash:         0v1f.hdig7.uavpc.3r4u4.kuioi.7tjfp.9lr4k.3p3i3.m6mc5.k14jf.m4kn2
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          remote
  source ship:      ~paldev
  source desk:      %verse
  source aeon:      11
  kids desk:        ~
  pending updates:  ~
::
  %groups
  /sys/kelvin:      [%zuse 417] [%zuse 416] [%zuse 415] [%zuse 414]
  base hash:        0v1e.6rrmc.iu8sv.dge36.i1bru.v6vcp.3o623.momic.1jjad.v0ah7.6ui9b
  %cz hash:         0v1e.6rrmc.iu8sv.dge36.i1bru.v6vcp.3o623.momic.1jjad.v0ah7.6ui9b
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  [~ ~sogryp-dister-dozzod-dozzod]
  updates:          remote
  source ship:      ~marnec-dozzod-marzod
  source desk:      %groups
  source aeon:      78
  kids desk:        ~
  pending updates:  ~
::
  %base
  /sys/kelvin:      [%zuse 414]
  base hash:        0vg.kniql.spcko.ms91o.v2qng.8ah7k.evj1b.sbcko.3deer.3f5eq.fk0pn
  %cz hash:         0vg.kniql.spcko.ms91o.v2qng.8ah7k.evj1b.sbcko.3deer.3f5eq.fk0pn
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          remote
  source ship:      ~marzod
  source desk:      %kids
  source aeon:      140
  kids desk:        %kids
  pending updates:  ~
::
  %page
  /sys/kelvin:      [%zuse 418]
  base hash:        0v5.aonh3.6uv5f.qfv94.2svjt.1bbr7.p7hv1.ks8ej.oblhd.qcv3m.j3pe6
  %cz hash:         0v5.aonh3.6uv5f.qfv94.2svjt.1bbr7.p7hv1.ks8ej.oblhd.qcv3m.j3pe6
  app status:       suspended
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          local
  source ship:      ~
  source desk:      ~
  source aeon:      ~
  kids desk:        ~
  pending updates:  ~
::
  %radio
  /sys/kelvin:      [%zuse 416] [%zuse 415] [%zuse 414]
  base hash:        0v3.36l8j.kpnl8.3qiij.9gai7.s1h9i.qvmpg.3qqs3.07g14.ah0mt.fvp5s
  %cz hash:         0v3.36l8j.kpnl8.3qiij.9gai7.s1h9i.qvmpg.3qqs3.07g14.ah0mt.fvp5s
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  [~ ~nodmyn-dosrux]
  updates:          remote
  source ship:      ~nodmyn-dosrux
  source desk:      %radio
  source aeon:      48
  kids desk:        ~
  pending updates:  ~
::
  %ccrur
  /sys/kelvin:      [%zuse 414]
  base hash:        0v1s.g24k5.i9vm1.mk228.gq9q9.6tb52.stj8b.nd21e.f7h75.4r0hd.8cavf
  %cz hash:         0v1s.g24k5.i9vm1.mk228.gq9q9.6tb52.stj8b.nd21e.f7h75.4r0hd.8cavf
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          remote
  source ship:      ~tondes-sitrym
  source desk:      %ccrur
  source aeon:      47
  kids desk:        ~
  pending updates:  ~
::
  %gora
  /sys/kelvin:      [%zuse 414]
  base hash:        0v1e.f0to9.s7a58.91vfb.s4rtg.rh8r9.72c4n.n4a4r.4pc5h.rbr3g.pfvo7
  %cz hash:         0v1e.f0to9.s7a58.91vfb.s4rtg.rh8r9.72c4n.n4a4r.4pc5h.rbr3g.pfvo7
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  [~ ~dister-dozzod-dalten]
  updates:          remote
  source ship:      ~dister-dozzod-dalten
  source desk:      %gora
  source aeon:      17
  kids desk:        ~
  pending updates:  ~
::
  %campfire
  /sys/kelvin:      [%zuse 415] [%zuse 414]
  base hash:        0v1n.57rvk.dul8r.m3gli.qu986.8g3dh.vlhje.m62uu.0qiv1.12ajo.rr9jj
  %cz hash:         0v1n.57rvk.dul8r.m3gli.qu986.8g3dh.vlhje.m62uu.0qiv1.12ajo.rr9jj
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  [~ ~dister-dister-datryn-ribdun]
  updates:          remote
  source ship:      ~dister-dister-datryn-ribdun
  source desk:      %campfire
  source aeon:      43
  kids desk:        ~
  pending updates:  ~
::
  %quorum
  /sys/kelvin:      [%zuse 416] [%zuse 415] [%zuse 414]
  base hash:        0v1c.edc18.t81t8.uo8up.9g0ce.en31t.rqtvm.jfbar.nndr1.tckve.fum2d
  %cz hash:         0vk.toe33.a89tm.63ads.3k74g.kktuj.k89o9.j580m.dqnfi.44o1e.8o46d
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          remote
  source ship:      ~dister-dister-sidnym-ladrut
  source desk:      %quorum
  source aeon:      11
  kids desk:        ~
  pending updates:  ~
::
  %sphinx
  /sys/kelvin:      [%zuse 417] [%zuse 416] [%zuse 415] [%zuse 414]
  base hash:        0v1a.4mmva.ud68e.7263t.tfecr.k7n6u.jvtfn.181u7.dgubd.uf7pr.q32nd
  %cz hash:         0v1a.4mmva.ud68e.7263t.tfecr.k7n6u.jvtfn.181u7.dgubd.uf7pr.q32nd
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          remote
  source ship:      ~dister-nocsyx-lassul
  source desk:      %sphinx
  source aeon:      19
  kids desk:        ~
  pending updates:  ~
::
  %cliff
  /sys/kelvin:      [%zuse 417] [%zuse 416] [%zuse 415] [%zuse 414]
  base hash:        0v1d.4grsd.c9c5h.uoj2s.33lgj.cr64j.itfhd.rasf7.muh0u.00mmo.hugl0
  %cz hash:         0v1d.4grsd.c9c5h.uoj2s.33lgj.cr64j.itfhd.rasf7.muh0u.00mmo.hugl0
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          remote
  source ship:      ~paldev
  source desk:      %cliff
  source aeon:      14
  kids desk:        ~
  pending updates:  ~
::
  %landscape
  /sys/kelvin:      [%zuse 416] [%zuse 415] [%zuse 414]
  base hash:        0v1t.835b9.armja.v46sm.0lao6.fqb2g.r132v.b87uj.j3t1h.4jaod.pd7qs
  %cz hash:         0v1t.835b9.armja.v46sm.0lao6.fqb2g.r132v.b87uj.j3t1h.4jaod.pd7qs
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  [~ ~lander-dister-dozzod-dozzod]
  updates:          remote
  source ship:      ~lander-dister-dozzod-dozzod
  source desk:      %landscape
  source aeon:      39
  kids desk:        ~
  pending updates:  ~
::
  %slam
  /sys/kelvin:      [%zuse 416]
  base hash:        0v1c.dauk0.k8c79.8imq1.k450b.v45me.jk4uu.4uah6.hrdp7.oksti.qm48c
  %cz hash:         0v1c.dauk0.k8c79.8imq1.k450b.v45me.jk4uu.4uah6.hrdp7.oksti.qm48c
  app status:       suspended
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          local
  source ship:      ~
  source desk:      ~
  source aeon:      ~
  kids desk:        ~
  pending updates:  ~
::
  %urdl
  /sys/kelvin:      [%zuse 414]
  base hash:        0v11.dlu32.j3vh7.rjegh.n15ko.m4t9c.s1aom.imaph.a0t45.sdaki.feflh
  %cz hash:         0v11.dlu32.j3vh7.rjegh.n15ko.m4t9c.s1aom.imaph.a0t45.sdaki.feflh
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  [~ ~dister-dozzod-dalten]
  updates:          remote
  source ship:      ~dister-dozzod-dalten
  source desk:      %urdl
  source aeon:      5
  kids desk:        ~
  pending updates:  ~
::
  %webterm
  /sys/kelvin:      [%zuse 416] [%zuse 415] [%zuse 414]
  base hash:        0v1c.r1qd1.7k3j2.1rmti.jq0tg.gpdp2.c2d58.npmcu.nndk1.j90ds.ap8ot
  %cz hash:         0v1c.r1qd1.7k3j2.1rmti.jq0tg.gpdp2.c2d58.npmcu.nndk1.j90ds.ap8ot
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  [~ ~mister-dister-dozzod-dozzod]
  updates:          remote
  source ship:      ~mister-dister-dozzod-dozzod
  source desk:      %webterm
  source aeon:      11
  kids desk:        ~
  pending updates:  ~
::
  %garden
  /sys/kelvin:      [%zuse 417] [%zuse 416] [%zuse 415] [%zuse 414]
  base hash:        0vg.7le9g.83mav.usq0n.2mi4c.biku1.engdt.c699u.71n6t.4d5bm.9ldns
  %cz hash:         0vg.7le9g.83mav.usq0n.2mi4c.biku1.engdt.c699u.71n6t.4d5bm.9ldns
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  [~ ~mister-dister-dozzod-dozzod]
  updates:          remote
  source ship:      ~mister-dister-dozzod-dozzod
  source desk:      %garden
  source aeon:      33
  kids desk:        ~
  pending updates:  ~
::
  %rumors
  /sys/kelvin:      [%zuse 417] [%zuse 416] [%zuse 415] [%zuse 414]
  base hash:        0v7.d3lj7.cpbdi.qpvbc.ssj3e.frpp4.6l7ff.24pb6.unk3q.00i9t.5vqbn
  %cz hash:         0v7.d3lj7.cpbdi.qpvbc.ssj3e.frpp4.6l7ff.24pb6.unk3q.00i9t.5vqbn
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          remote
  source ship:      ~paldev
  source desk:      %rumors
  source aeon:      15
  kids desk:        ~
  pending updates:  ~
::
  %nectar
  /sys/kelvin:      [%zuse 417] [%zuse 416] [%zuse 415] [%zuse 414]
  base hash:        0vv.pk4b7.bdeb2.idgq1.n5mtd.jkoo3.kukq4.l1132.25643.5nf0i.t47uo
  %cz hash:         0v6.b5mdi.hevql.qosva.s27p0.rpjdv.rrk1c.mft92.36e5s.ljbbi.7v2s0
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  [~ ~bacrys]
  updates:          remote
  source ship:      ~bacrys
  source desk:      %nectar
  source aeon:      12
  kids desk:        ~
  pending updates:  ~
::
  %sentinel
  /sys/kelvin:      [%zuse 416] [%zuse 415] [%zuse 414]
  base hash:        0vp.jstht.bkkpn.g1od3.2vn3j.cirhb.5578p.15cc3.8qq2c.fha0j.jp8u7
  %cz hash:         0vp.jstht.bkkpn.g1od3.2vn3j.cirhb.5578p.15cc3.8qq2c.fha0j.jp8u7
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          remote
  source ship:      ~dister-dozzod-lapdeg
  source desk:      %sentinel
  source aeon:      16
  kids desk:        ~
  pending updates:  ~
::
  %docs
  /sys/kelvin:      [%zuse 417] [%zuse 416] [%zuse 415] [%zuse 414]
  base hash:        0v1r.dpmuq.88v40.db4ge.5931k.srq21.ac0lo.bih1e.0hsha.les12.8gg87
  %cz hash:         0v1r.dpmuq.88v40.db4ge.5931k.srq21.ac0lo.bih1e.0hsha.les12.8gg87
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          remote
  source ship:      ~pocwet
  source desk:      %docs
  source aeon:      36
  kids desk:        ~
  pending updates:  ~
::
  %astrolabe
  /sys/kelvin:      [%zuse 414]
  base hash:        0v1n.u188l.0fae5.g6abd.sq57k.qllrv.h4dpo.psuao.kc7qr.kgkm9.trmfm
  %cz hash:         0v1n.u188l.0fae5.g6abd.sq57k.qllrv.h4dpo.psuao.kc7qr.kgkm9.trmfm
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          remote
  source ship:      ~dister-midlev-mindyr
  source desk:      %astrolabe
  source aeon:      62
  kids desk:        ~
  pending updates:  ~
::
  %trill
  /sys/kelvin:      [%zuse 418]
  base hash:        0v13.v33n9.t5nq0.a2geg.bmvrr.111ad.q76ft.t44rc.nbubl.r0are.pb15b
  %cz hash:         0v13.v33n9.t5nq0.a2geg.bmvrr.111ad.q76ft.t44rc.nbubl.r0are.pb15b
  app status:       suspended
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          local
  source ship:      ~
  source desk:      ~
  source aeon:      ~
  kids desk:        ~
  pending updates:  ~
::
  %templeochess
  /sys/kelvin:      [%zuse 417]
  base hash:        0v1r.urtss.ferl3.0lasg.8qvih.ts0i1.bmmf0.igcdv.pa7vg.eolo5.91am2
  %cz hash:         0v1r.urtss.ferl3.0lasg.8qvih.ts0i1.bmmf0.igcdv.pa7vg.eolo5.91am2
  app status:       suspended
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          local
  source ship:      ~
  source desk:      ~
  source aeon:      ~
  kids desk:        ~
  pending updates:  ~
::
  %chess
  /sys/kelvin:      [%zuse 414]
  base hash:        0v2.kbr58.2bla5.k67pd.gf1g4.sts06.sjsap.1gmh7.ei34t.qhof8.hfhud
  %cz hash:         0v2.kbr58.2bla5.k67pd.gf1g4.sts06.sjsap.1gmh7.ei34t.qhof8.hfhud
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  [~ ~finmep-lanteb]
  updates:          remote
  source ship:      ~finmep-lanteb
  source desk:      %chess
  source aeon:      31
  kids desk:        ~
  pending updates:  ~
::
  %ballot
  /sys/kelvin:      [%zuse 417] [%zuse 416] [%zuse 415]
  base hash:        0v1a.qao47.ui9eb.ki3si.u129b.r5ra1.voel8.ucc5d.t9vl2.629c9.lm3ru
  %cz hash:         0v1a.qao47.ui9eb.ki3si.u129b.r5ra1.voel8.ucc5d.t9vl2.629c9.lm3ru
  app status:       suspended
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          local
  source ship:      ~
  source desk:      ~
  source aeon:      ~
  kids desk:        ~
  pending updates:  ~
::
  %atlas
  /sys/kelvin:      [%zuse 418]
  base hash:        0v1n.bjf44.qg3hb.tu3ai.nls93.rna8u.60c3f.seqq9.u1epm.dm01c.bk468
  %cz hash:         0v1n.bjf44.qg3hb.tu3ai.nls93.rna8u.60c3f.seqq9.u1epm.dm01c.bk468
  app status:       suspended
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          local
  source ship:      ~
  source desk:      ~
  source aeon:      ~
  kids desk:        ~
  pending updates:  ~
::
  %pals
  /sys/kelvin:      [%zuse 417] [%zuse 416] [%zuse 415] [%zuse 414]
  base hash:        0v1u.3iaik.f1r49.9flat.pe2br.7g4j0.7secj.9leoq.idqdc.emp85.vfhmk
  %cz hash:         0v1u.3iaik.f1r49.9flat.pe2br.7g4j0.7secj.9leoq.idqdc.emp85.vfhmk
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          remote
  source ship:      ~paldev
  source desk:      %pals
  source aeon:      28
  kids desk:        ~
  pending updates:  ~
::
  %ufa
  /sys/kelvin:      [%zuse 418]
  base hash:        0v1f.3knok.r0o4d.j6kqi.e7cab.vfo3i.on7ip.h3d5a.vh4fe.86mq8.gmiar
  %cz hash:         0v1f.3knok.r0o4d.j6kqi.e7cab.vfo3i.on7ip.h3d5a.vh4fe.86mq8.gmiar
  app status:       suspended
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          local
  source ship:      ~
  source desk:      ~
  source aeon:      ~
  kids desk:        ~
  pending updates:  ~
::
  %houston
  /sys/kelvin:      [%zuse 416] [%zuse 415]
  base hash:        0v2.9cvke.5np43.4bqvu.nigtq.cg0l9.0kev0.7f4s2.go4sq.84oa2.oaqp6
  %cz hash:         0v2.9cvke.5np43.4bqvu.nigtq.cg0l9.0kev0.7f4s2.go4sq.84oa2.oaqp6
  app status:       suspended
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          local
  source ship:      ~
  source desk:      ~
  source aeon:      ~
  kids desk:        ~
  pending updates:  ~
::
  %board
  /sys/kelvin:      [%zuse 414]
  base hash:        0v1f.q3i6g.havbd.014bq.agniq.0qq5o.a3hj7.5ghra.kpota.c4ha9.mhcdc
  %cz hash:         0v1f.q3i6g.havbd.014bq.agniq.0qq5o.a3hj7.5ghra.kpota.c4ha9.mhcdc
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  [~ ~dister-hanfel-dovned]
  updates:          remote
  source ship:      ~dister-hanfel-dovned
  source desk:      %board
  source aeon:      18
  kids desk:        ~
  pending updates:  ~
::
  %dukebox
  /sys/kelvin:      [%zuse 415]
  base hash:        0v1k.ofcmr.18b5i.ql596.1574d.s2sbo.ik0q3.6sj31.kggst.5j6a0.cuihb
  %cz hash:         0v1k.ofcmr.18b5i.ql596.1574d.s2sbo.ik0q3.6sj31.kggst.5j6a0.cuihb
  app status:       suspended until next update
  force on:         ~
  force off:        ~
  publishing ship:  [~ ~zod]
  updates:          remote
  source ship:      ~tagrev-lacmur-lomped-firser
  source desk:      %dukebox
  source aeon:      16
  kids desk:        ~
  pending updates:  ~
::
  %feature
  /sys/kelvin:      [%zuse 418]
  base hash:        0vs.nam16.vhood.9d556.8s133.1rgaa.5a4r1.7dqda.3uhg1.g45os.3rjqp
  %cz hash:         0vs.nam16.vhood.9d556.8s133.1rgaa.5a4r1.7dqda.3uhg1.g45os.3rjqp
  app status:       suspended
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          local
  source ship:      ~
  source desk:      ~
  source aeon:      ~
  kids desk:        ~
  pending updates:  ~
::
  %gol-cli
  /sys/kelvin:      [%zuse 417]
  base hash:        0v2.gbgei.53bv2.td1pn.r7utn.er9vh.r3hcf.ggl8c.114hu.9kalb.jdcp9
  %cz hash:         0v2.gbgei.53bv2.td1pn.r7utn.er9vh.r3hcf.ggl8c.114hu.9kalb.jdcp9
  app status:       suspended
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          local
  source ship:      ~
  source desk:      ~
  source aeon:      ~
  kids desk:        ~
  pending updates:  ~
::
  %portal
  /sys/kelvin:      [%zuse 417] [%zuse 416] [%zuse 415] [%zuse 414]
  base hash:        0v16.gq8sp.h54dl.drhku.isgj6.gbajb.uu9g9.nb8lc.dgnbf.0sclb.970l3
  %cz hash:         0v16.gq8sp.h54dl.drhku.isgj6.gbajb.uu9g9.nb8lc.dgnbf.0sclb.970l3
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  [~ ~worpet-bildet]
  updates:          remote
  source ship:      ~worpet-bildet
  source desk:      %portal
  source aeon:      43
  kids desk:        ~
  pending updates:  ~
::
  %peat
  /sys/kelvin:      [%zuse 417]
  base hash:        0v10.nqp22.nrk8i.oeisk.a3qom.0dhth.18a6e.5j0v8.9r3fr.smhl2.pdl07
  %cz hash:         0v10.nqp22.nrk8i.oeisk.a3qom.0dhth.18a6e.5j0v8.9r3fr.smhl2.pdl07
  app status:       suspended until next update
  force on:         ~
  force off:        ~
  publishing ship:  [~ ~dister-dozzod-dalten]
  updates:          remote
  source ship:      ~dister-dozzod-dalten
  source desk:      %peat
  source aeon:      10
  kids desk:        ~
  pending updates:  ~
::
  %channel
  /sys/kelvin:      [%zuse 418]
  base hash:        0vj.t4jr8.nenri.9qv7c.2cf5q.8r3la.d5ls4.liap4.43qlo.bleeq.bsakt
  %cz hash:         0vj.t4jr8.nenri.9qv7c.2cf5q.8r3la.d5ls4.liap4.43qlo.bleeq.bsakt
  app status:       suspended
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          local
  source ship:      ~
  source desk:      ~
  source aeon:      ~
  kids desk:        ~
  pending updates:  ~
::
  %aera
  /sys/kelvin:      [%zuse 414]
  base hash:        0ve.uqhva.n3p7l.7thp4.h395i.hljg9.lat5h.u4bhm.hasjd.5uueo.i6dda
  %cz hash:         0ve.uqhva.n3p7l.7thp4.h395i.hljg9.lat5h.u4bhm.hasjd.5uueo.i6dda
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  [~ ~dister-dozzod-dalten]
  updates:          remote
  source ship:      ~dister-dozzod-dalten
  source desk:      %aera
  source aeon:      8
  kids desk:        ~
  pending updates:  ~
::
  %scooore
  /sys/kelvin:      [%zuse 417] [%zuse 416] [%zuse 415] [%zuse 414]
  base hash:        0vn.4ib8h.egsko.6o3sl.blj9k.ap8fh.9r8ta.4a23g.gkoiu.epvm0.aab0n
  %cz hash:         0vn.4ib8h.egsko.6o3sl.blj9k.ap8fh.9r8ta.4a23g.gkoiu.epvm0.aab0n
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          remote
  source ship:      ~paldev
  source desk:      %scooore
  source aeon:      4
  kids desk:        ~
  pending updates:  ~
::
 %kids %kids %cz hash:     0vg.kniql.spcko.ms91o.v2qng.8ah7k.evj1b.sbcko.3deer.3f5eq.fk0pn::
  %escape
  /sys/kelvin:      [%zuse 417]
  base hash:        0v1n.7gt06.2tbpb.59e1u.1hrj4.7esau.kgm8k.hcgcd.5jq66.c5jrt.d0u8h
  %cz hash:         0v1n.7gt06.2tbpb.59e1u.1hrj4.7esau.kgm8k.hcgcd.5jq66.c5jrt.d0u8h
  app status:       suspended
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          local
  source ship:      ~
  source desk:      ~
  source aeon:      ~
  kids desk:        ~
  pending updates:  ~
::
  %talk
  /sys/kelvin:      [%zuse 417] [%zuse 416] [%zuse 415] [%zuse 414]
  base hash:        0v17.0vdhg.afr4a.eda6m.lrjkv.qru36.63ds1.15uhi.6v0ih.qfp9n.v6bka
  %cz hash:         0v17.0vdhg.afr4a.eda6m.lrjkv.qru36.63ds1.15uhi.6v0ih.qfp9n.v6bka
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  [~ ~sogryp-dister-dozzod-dozzod]
  updates:          remote
  source ship:      ~marnec-dozzod-marzod
  source desk:      %talk
  source aeon:      74
  kids desk:        ~
  pending updates:  ~
::
```

**Comment by patosullivan** [comment]
The odd thing about this error is that the stack trace includes react-beautiful-dnd, which isn't used in the grid component anymore. Can you try syncing %talk directly to ~sogryp-dister-dozzod-dozzod? `|install ~sogryp-dister-dozzod-dozzod %talk`? Probably a good idea to do the same with groups.

I'm not sure why the glob on marnec would include the old grid component either.

Which browser are you using?

**Comment by patosullivan** [comment]
Ok, the react-beautiful-dnd bit appears to be a red herring after looking into it. Still interested to see if switching source ships does anything for you on this one.

**Comment by bonbud-macryg** [comment]
Original error was in Arc, which is Chromium-based. I tried opening the Apps menu in Safari and got the error "Right side of assignment cannot be destructured", with this stack trace:

```
ae@http://100.93.206.31/apps/talk/assets/grid.3f745d3b.js:1:1600
Ko@http://100.93.206.31/apps/talk/assets/react-beautiful-dnd.a94a7f90.js:19:52866
ea@http://100.93.206.31/apps/talk/assets/react-beautiful-dnd.a94a7f90.js:23:8697
Js@http://100.93.206.31/apps/talk/assets/react-beautiful-dnd.a94a7f90.js:23:937
Qf@http://100.93.206.31/apps/talk/assets/react-beautiful-dnd.a94a7f90.js:23:868
sn@http://100.93.206.31/apps/talk/assets/react-beautiful-dnd.a94a7f90.js:23:727
mo@http://100.93.206.31/apps/talk/assets/react-beautiful-dnd.a94a7f90.js:21:10865
mo@[native code]
@http://100.93.206.31/apps/talk/assets/react-beautiful-dnd.a94a7f90.js:19:38698
@http://100.93.206.31/apps/talk/assets/react-beautiful-dnd.a94a7f90.js:8:3836
_s@http://100.93.206.31/apps/talk/assets/react-beautiful-dnd.a94a7f90.js:19:38645
ll@http://100.93.206.31/apps/talk/assets/react-beautiful-dnd.a94a7f90.js:8:2875
@http://100.93.206.31/apps/talk/assets/react-beautiful-dnd.a94a7f90.js:8:1547
```

I'll switch OTA sources and report back.

**Comment by patosullivan** [comment]
Fixed by #2483
